[
  {
    "DefinitionID": "oval:com.ubuntu.xenial:def:2020100810000000",
    "Title": "CVE-2020-10081 on Ubuntu 16.04 LTS (xenial) - medium.",
    "Description": "GitLab before 12.8.2 has Incorrect Access Control. It was internally discovered that the LFS import process could potentially be used to incorrectly access LFS objects not owned by the user.",
    "Advisory": {
      "Severity": "Medium",
      "Cves": [
        {
          "CveID": "CVE-2020-10081",
          "Cvss2": "",
          "Cvss3": "",
          "Cwe": "",
          "Impact": "",
          "Href": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-10081",
          "Public": ""
        }
      ],
      "Bugzillas": [],
      "AffectedCPEList": [],
      "Issued": "1000-01-01T00:00:00Z",
      "Updated": "1000-01-01T00:00:00Z"
    },
    "Debian": null,
    "AffectedPacks": [],
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2020-10081",
        "RefURL": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-10081"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "http://people.canonical.com/~ubuntu-security/cve/2020/CVE-2020-10081.html"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "https://about.gitlab.com/releases/2020/03/04/gitlab-12-dot-8-dot-2-released/"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "https://about.gitlab.com/releases/2020/03/04/gitlab-12-dot-8-dot-2-released/index.html"
      }
    ]
  }
]
